### PR TITLE
[CEN-362] App insights conf

### PR DESCRIPTION
**Pull request by pasqualedevita**
### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by uolter — APPROVED** [review]

File: src/aks/subscriptions/PROD-CSTAR/terraform.tfvars [hunk 1]
```diff
@@ -7,60 +7,78 @@ ingress_load_balancer_ip = "x.x.x.x"
...
+  BPD_ENROLLMENT_SERVER_THREAD_MAX                  = "400"
 }
 
 # bpdmsnotificationmanager
 configmaps_bpdmsnotificationmanager = {
+  JAVA_TOOL_OPTIONS                                                = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                                    = "bpdmsnotificationmanager"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL                = "OFF"
   BPD_AWARD_PERIOD_HOST                                            = "bpdmsawardperiod"
   JAVA_TOOL_OPTIONS                                                = "-Xms256m -Xmx6g"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/PROD-CSTAR/terraform.tfvars:83]: Here you need to have only one JAVA_TOOL_OPTIONS "-Xms256m -Xmx6g -javaagent:/applicationinsights-agent-3.1.0.jar"

File: src/aks/subscriptions/PROD-CSTAR/terraform.tfvars [hunk 2]
```diff
@@ -165,29 +198,38 @@ configmaps_bpdmsrankingprocessorpoc = {
...
 # rtdpaymentinstrumentmanager
 configmaps_rtdpaymentinstrumentmanager = {
-  JAVA_TOOL_OPTIONS                        = "-Xms128m -Xmx6g"
-  POSTGRES_POOLSIZE                        = "2"
-  POSTGRES_SHOW_SQL                        = "false"
-  LOG_LEVEL_RTD_PAYMENT_INSTRUMENT_MANAGER = "INFO"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                     = "rtdpaymentinstrumentmanager"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL = "OFF"
+  JAVA_TOOL_OPTIONS                                 = "-Xms128m -Xmx6g"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/PROD-CSTAR/terraform.tfvars:230]: ditto

File: src/aks/subscriptions/UAT-CSTAR/terraform.tfvars [hunk 3]
```diff
@@ -7,64 +7,82 @@ ingress_load_balancer_ip = "x.x.x.x"
...
+  BPD_ENROLLMENT_SERVER_THREAD_MAX                  = "400"
 }
 
 # bpdmsnotificationmanager
 configmaps_bpdmsnotificationmanager = {
+  JAVA_TOOL_OPTIONS                                        = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                            = "bpdmsnotificationmanager"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL        = "OFF"
   BPD_AWARD_PERIOD_HOST                                    = "bpdmsawardperiod"
   JAVA_TOOL_OPTIONS                                        = "-Xms256m -Xmx6g"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/UAT-CSTAR/terraform.tfvars:87]: ditto

File: src/aks/subscriptions/UAT-CSTAR/terraform.tfvars [hunk 4]
```diff
@@ -194,9 +232,12 @@ configmaps_bpdmswinningtransaction = {
...
 # rtdpaymentinstrumentmanager
 configmaps_rtdpaymentinstrumentmanager = {
-  JAVA_TOOL_OPTIONS                        = "-Xms128m -Xmx2g"
-  POSTGRES_POOLSIZE                        = "5"
-  POSTGRES_SHOW_SQL                        = "false"
-  LOG_LEVEL_RTD_PAYMENT_INSTRUMENT_MANAGER = "INFO"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                     = "rtdpaymentinstrumentmanager"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL = "OFF"
+  JAVA_TOOL_OPTIONS                                 = "-Xms128m -Xmx2g"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/UAT-CSTAR/terraform.tfvars:238]: ditto

File: src/aks/subscriptions/DEV-CSTAR/terraform.tfvars [hunk 5]
```diff
@@ -135,31 +171,39 @@ configmaps_bpdmsrankingprocessorpoc = {
...
+  WINN_TRX_SECONDARY_DB_ENABLE                      = "true"
 }
 
 # rtdpaymentinstrumentmanager
 configmaps_rtdpaymentinstrumentmanager = {
-  JAVA_TOOL_OPTIONS                        = "-Xms128m -Xmx2g -javaagent:/applicationinsights-agent-3.1.0.jar"
-  POSTGRES_POOLSIZE                        = "2"
-  POSTGRES_SHOW_SQL                        = "true"
-  LOG_LEVEL_RTD_PAYMENT_INSTRUMENT_MANAGER = "INFO"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/DEV-CSTAR/terraform.tfvars:202]: Duplicate

File: src/aks/subscriptions/PROD-CSTAR/terraform.tfvars (hunk 1 above)
Comment (pasqualedevita) [reply src/aks/subscriptions/PROD-CSTAR/terraform.tfvars:83]: ops copy and paste <3

File: src/aks/subscriptions/UAT-CSTAR/terraform.tfvars [hunk 6]
```diff
@@ -7,66 +7,83 @@ ingress_load_balancer_ip = "x.x.x.x"
 
 # bpdmsawardperiod
 configmaps_bpdmsawardperiod = {
-  AWARD_PERIOD_POOLSIZE              = "20"
-  AWARD_PERIOD_READ_ONLY             = "true"
-  BPD_AWARD_PERIOD_SERVER_THREAD_MAX = "350"
-  LOG_LEVEL_BPD_AWARD_PERIOD         = "INFO"
-  POSTGRES_SHOW_SQL                  = "false"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                     = "configmaps_bpdmsawardperiod"
```
Comment (ElfoLiNk) [inline src/aks/subscriptions/UAT-CSTAR/terraform.tfvars:11]: typo

**Review by ElfoLiNk — APPROVED** [review]

**Pull request merged**